Xamarin.Forms.Platform.GTK/Controls/Page.cs

```diff
@@ -135,15 +140,15 @@ private void BuildPage()
 
 		private void RefreshToolbar(HBox newToolbar)
 		{
-			_headerContainer.RemoveFromContainer(_toolbar);
```

Comment:
Why we don't need to call RemoveFromContainer in the HeaderContainer? it is part of the _toolbar?

Reply:
Destroy() already remove it from the container